### ISSUE TITLE
TARGET_TIMESPAN

### ISSUE THREAD

**Issue by bigdaub** [hank/feathercoinj#1]
update NetworkParameters Line 281 with  (7 \* 24 \* 60 \* 60) / 8

**Comment by hank** [comment]
Yeah I did this a while back and haven't committed it.  I'll get around to that soon - sorry for the delay!

**Comment by bigdaub** [comment]
No problem. I saw this when converting to bbq and wanted to note it before I forgot.
